<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctgadminsaveDialog</class>
 <widget class="QDialog" name="ctgadminsaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 0, 0)</string>
  </property>
  <widget class="QLabel" name="label_namec">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>    Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_descriptionc">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>  description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_indentityc">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>   identity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eidc">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>     eid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_namec">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>   Agrabad Resort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_descriptionc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>  Agrabad,Chittagong</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_identityc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>   book or booked</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eidc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>  ctg01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savec">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>330</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background:rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updatec">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background:rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
